src/schema-templates/recipe-description.block.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Recipe description block schema template.
+ *
+ * @package Yoast\WP\SEO\Schema_Templates
+ */
+
+use Yoast\WP\SEO\Schema_Templates\Assets\Icons;
+
+// phpcs:disable WordPress.Security.EscapeOutput -- Reason: The Icons contains safe svg.
+?>
+{{block name="yoast/recipe-description" title="<?php esc_attr_e( 'Recipe description', 'wordpress-seo-premium' ); ?>" category="yoast-required-recipe-blocks" description="<?php esc_attr_e( 'The description of the recipe.', 'wordpress-seo-premium' ); ?>" icon="<?php echo Icons::heroicons_identification(); ?>" supports={"multiple": false} }}
```

Comment:
You should use FQN global function calls in namespaced files (i.e., use `\` before WordPress and PHP functions). Please check all occurrences.

Reply:
Shouldn't there be a PHPCS sniff for that?

Reply:
That would be very helpful... @jrfnl do you know if this is being worked on?

Reply:
There is a sniff (which I use locally), but it's from an external standard which has a PHP 7.1 minimum PHP version, which is why we can't add it to YoastCS, as it would conflict with the minimum PHP version of the plugins.

Reply:
Ah, thanks @jrfnl! 💡